package.json

```diff
@@ -1,23 +1,68 @@
 {
+  "name": "headset",
+  "productName": "Headset",
   "version": "2.2.0",
+  "description": "Discover and collect music on YouTube",
+  "license": "MIT",
+  "author": "Daniel Ravina <hello@headsetapp.co>",
+  "homepage": "http://headsetapp.co",
+  "bugs": "https://github.com/headsetapp/headset-electron/issues",
+  "main": "src/index.js",
+  "private": true,
   "scripts": {
-    "lint": "eslint",
-    "version": "npm-publish-all bump --bumpType=$npm_package_version && npm run bumps && git add .",
+    "headset": "electron .",
+    "player": "bash bin/player_window.sh",
```

Comment:
As we have many scripts now, I'm thinking maybe it's a good opportunity to remove the `headset` and `player` scripts and change the `Procfile` (run by nf) to: 
```
headset: ./node_modules/.bin/electron .
player: bash bin/player_window.sh
```

to my knowledge, they only run by nf in development, this will remove ambiguity from the script

Reply:
I think that removing the `player` script is a good idea. That's only used by `nf`. I don't think it's a good idea to remove the `headset` script. I regularly use that without `nf`, to test the wrapper with the current core loaded from the official url, so I don't have to run the local core server. But since we do have a lot of scripts, we can run that with `npx electron .` and mention that on all the docs. Thoughts?

Reply:
nah.. if this requires changes in the docs than there's no point.

Another way is to maybe prefix the script name with `dev:` to be consistent with the `pack:` and `dist:` 
```
npm run dev:headset 
```

Reply:
good tip about `npx`. I feel like I should have known about it. I'm so used to ./node_modules/.bin. never thought of changing the old ways